egs/librispeech/ASR/conformer_ctc/label_smoothing.py

```diff
@@ -45,6 +45,7 @@ def __init__(
         """
         super().__init__()
         assert 0.0 <= label_smoothing < 1.0
+        assert reduction in ("none", "sum", "mean")
```

Comment:
```suggestion
        assert reduction in ("none", "sum", "mean"), reduction
```
It is helpful to print out the value of reduction on assertion failure.